src/GraphQL/Types/Scalars/UIntGraphType.cs

```diff
@@ -23,10 +23,12 @@ public class UIntGraphType : ScalarGraphType
         /// <inheritdoc/>
         public override bool CanParseLiteral(IValue value) => value switch
         {
-            UIntValue _ => true,
             IntValue intValue => intValue.Value >= 0,
             LongValue longValue => uint.MinValue <= longValue.Value && longValue.Value <= uint.MaxValue,
             _ => false
         };
+
+        /// <inheritdoc/>
+        public override IValue ToAST(object value) => new LongValue(Convert.ToUInt32(value));
```

Comment:
Always `LongValue` ? Maybe use `IntValue` when value is in `Int` range ?

Reply:
Why?

* This code is used to return default values via introspection, nothing else presently
* LongValue is a perfectly legal AST representation of a `uint`
* This is not a spot for speed optimization, and adding a switch would not improve speed
* It would simply add additional lines of code to the codebase.

It would make more sense to allow `IntValue.ParseLiteral` to accept `LongValue` and `BigIntValue`, since they are legal representations of an integer, even if normally they are only used for values outside the range of `int`.

Reply:
I'm not sure here.

> This code is used to return default values via introspection, nothing else presently

Are you 100% sure ? I don't remember. 

> It would make more sense to allow IntValue.ParseLiteral to accept LongValue and BigIntValue, since they are legal representations of an integer, even if normally they are only used for values outside the range of int.

Good.

Reply:
Yes, positive. 

Ok I’ll make the changes.